seeds/tutorSeeds.json

```diff
@@ -1,6 +1,38 @@
 [
     {
-        "name": "Anthony Brown",
-        "email": "password12345@gmail.com"
+        "name": "Dwight Schrute", 1
```

Comment:
The numbers at the end are going to cause errors because this is invalid JSON syntax. Were you thinking these would be the id numbers?

You can fix it by moving the numbers up to a separate key: value pair like:

`"id": 1,`